docs/First-Installation.md

```diff
@@ -58,44 +58,40 @@ The simplest way to get most of these components is through an all-in-one kit su
    * For macOS: [Raspberry Pi Imager](https://www.raspberrypi.org/software/), [Etcher](https://www.balena.io/etcher/), [Apple Pi Baker](https://www.tweaking4all.com/hardware/raspberry-pi/applepi-baker-v2/), or the `dd` command
    * For Linux: [Raspberry Pi Imager](https://www.raspberrypi.org/software/), [Etcher](https://www.balena.io/etcher/), or the `dd` command
     * MacOS/Linux users can optionally extract the .img image from the downloaded .img.gz by using `gunzip` (macOS users can also simply double-click it)
-4.  Once the program's image/OS and SD card have been chosen, *write* it to the SD card and wait until the operation completes
-5.  Remove the MicroSD card from your computer, slide it back into the slot on your Raspberry Pi, and turn it on
+4. Once the program's image/OS and SD card have been chosen, *write* it to the SD card and wait until the operation completes
+5. Remove the MicroSD card from your computer, slide it back into the slot on your Raspberry Pi
+6. Plug in a controller or keyboard
```

Comment:
do we really need this step... come on :P

Reply:
I can't tell which line you've selected. Line 63? Until this point it's never specified to plug in the controller, and it's crucial to get the next part working with controllers. Thought it couldn't hurt.

Reply:
I don't recall ever seeing come for support for this, so it's noise IMO